package.json

```diff
@@ -11,7 +11,7 @@
     "build:backend": "yarn workspace backend build",
     "build:all": "backstage-cli repo build --all",
     "build:api-reports": "yarn build:api-reports:only --tsc",
-    "build:api-reports:only": "backstage-repo-tools api-reports",
+    "build:api-reports:only": "backstage-repo-tools api-reports --allow-warnings packages/core-components  plugins/catalog plugins/catalog-import plugins/git-release-manager plugins/jenkins plugins/kubernetes",
```

Comment:
This is confusing me a bit, is it building API reports for `plugins/catalog` and all the following paths, or allowing warnings for them? I would probable expect something like `--allow-warnings path1,path2,path3` or `--allow-warnings path1 --allow-warnings path2` if that wasn't the case.

Reply:
good point, yes is how commander.js proposed, and didn’t think about it.
- this is work as is`--allow-warnings path1 --allow-warnings path2` 
- this is easy to implement too `--allow-warnings path1,path2,path3` will give this a try
- we are accepting glob patterns for the arguments, maybe can also allow glob here to make it 
`backstage-repo-tools api-reports --allow-warnings packages/core-components --allow-warnings  plugins/(catalog|catalog-import|git-release-manager|jenkins|kubernetes)`

WDYT?

Reply:
looking at [this](https://github.com/tj/commander.js/blob/master/docs/options-taking-varying-arguments.md#alternative-use-options-instead-of-command-arguments), seems a good idea to use the path as an option here instead of argument I'm working on this now too

Reply:
implemented 2 and 3, the `,` will be processed always first, and then the glob processed for each element.

one issue with the `,` values is that `-a packages/*,plugins/catalog*` will throw an error on the terminal as `zsh` that tries to resolve glob patterns and need to be passed as a string `-a 'packages/*,plugins/catalog*'`